### PR TITLE
Specify net-http instead of httpclient in header

### PR DESCRIPTION
**Pull request by phallobst**

### CODE REVIEW

**Comment by Carlgo11** [comment]
`Gem.loaded_specs['net-http']` just returns nil for me

File: tests/language-codes.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
   headers = {
     'Accept' => 'application/json',
     'User-Agent' => '2FactorAuth/LanguageValidator '\
-    "(HTTPClient/#{Gem.loaded_specs['httpclient'].version} on Ruby/#{RUBY_VERSION}; +https://2fa.directory/bot)",
+    "(Net::HTTP/#{Gem.loaded_specs['net-http'].version} on Ruby/#{RUBY_VERSION}; +https://2fa.directory/bot)",
```
Comment (Carlgo11) [inline tests/language-codes.rb]:
```suggestion
    "(Ruby/#{RUBY_VERSION}; +https://2fa.directory/bot)",
```

File: tests/region-codes.rb [hunk 2]
```diff
@@ -16,7 +16,7 @@
   headers = {
     'Accept' => 'application/json',
     'User-Agent' => '2FactorAuth/RegionValidator' \
-    "(HTTPClient/#{Gem.loaded_specs['httpclient'].version} on Ruby/#{RUBY_VERSION}; +https://2fa.directory/bot)",
+    "(Net::HTTP/#{Gem.loaded_specs['net-http'].version} on Ruby/#{RUBY_VERSION}; +https://2fa.directory/bot)",
```
Comment (Carlgo11) [inline tests/region-codes.rb]:
```suggestion
    "(Ruby/#{RUBY_VERSION}; +https://2fa.directory/bot)",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
> `Gem.loaded_specs['net-http']` just returns nil for me

That's basically how I stumbled upon this problem - the `httpclient` gem was not installed and the script gave me an error message. Does the test work for you at all, @Carlgo11? (Without the cached file in `/tmp`, that is.)

**Comment by Carlgo11** [comment]
Script works fine without `Gem.loaded_specs['net-http']`.
httpclient is also in the tests group so running `bundle install --with tests` should install it for you.

**Comment by phallobst** [comment]
Since these tests use Net::HTTP instead of HTTPClient, I figured it would be useless to specify the `httpclient` version, but leaving it out altogether is okay for me as well. :man_shrugging:

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Net is part of the core Ruby methods so it makes sense leaving it out.

**Pull request merged**